armbian/base/build/customize-armbian-rockpro64.sh

```diff
@@ -369,15 +367,35 @@ EOF
 
 
 # LIGHTNING --------------------------------------------------------------------
+BIN_DEPS_TAG="v0.0.1-alpha"
 LIGHTNING_VERSION="0.7.0"
 
-cd /usr/local/src/
-git clone https://github.com/ElementsProject/lightning.git || true
-cd lightning
-git checkout v${LIGHTNING_VERSION}
-./configure
-make
-make install
+## either compile c-lightning from source (default), or use prebuilt binary
+if [ "${BASE_BUILD_LIGHTNINGD}" == "true" ]; then
+  apt install -y  autoconf automake build-essential git libtool libgmp-dev \
+                  libsqlite3-dev python python3 net-tools zlib1g-dev libsodium-dev
+
+  cd /usr/local/src/
+  git clone https://github.com/ElementsProject/lightning.git || true
+  cd lightning
+  git checkout v${LIGHTNING_VERSION}
+  ./configure
+  make -j 4
+  make install
+
+else
+  apt install -y libsodium-dev
+  cd /usr/local/src/
+  # temporary storage of 'lightningd' until official arm64 binaries work with stable Armbian release
+  curl --retry 5 -SLO https://github.com/digitalbitbox/bitbox-base-deps/releases/download/${BIN_DEPS_TAG}/lightningd_${LIGHTNING_VERSION}-1_arm64.deb
+  if ! echo "52be094f8162749acb207bf9ad08125d25288a9d03eb25690f364ba42fcff3c4  lightningd_0.7.0-1_arm64.deb" | sha256sum -c -; then
+    echo "sha256sum for precompiled 'lightningd' failed" >&2
+    exit 1
+  fi
+  dpkg -i lightningd_${LIGHTNING_VERSION}-1_arm64.deb
+  ln -s /usr/bin/lightningd /usr/local/bin/lightningd
```

Comment:
I'm curious why this symlink is needed.. is the `make install` going to put `lightningd` in a different path than installing from the `.deb` package?

Reply:
Yes, exactly. I'll add a comment. The `make install` command puts the binaray in `/usr/local/bin`, while the deb package installs into `/usr/bin`...

Reply:
Cool, adding a comment sounds useful then.